ang/crmCaseType.js

```diff
@@ -300,16 +308,25 @@
             });
           }
         }
-        // TODO The ids below really should use names not labels see
-        //  https://lab.civicrm.org/dev/core/issues/774
+        // TODO The `id` value is caught in between trying to support xml
+        // files where using the relType.id doesn't make it portable between
+        // installs and the xml doesn't have a concept of direction, but
+        // using name_a_b would conflict with it being used currently to set
+        // displaylabel elsewhere. So for now `id` is label.
         else {
           result.push({
+            xmlNameOtherDirection: relType.name_a_b,
+            xmlName: relType.name_b_a,
```

Comment:
Ok I think these values got mixed up in a few places when re-labeling them.  I think these options are supposed to have the following pattern:

|   | `a_b` (the non-client's perspective, as in XML) | `b_a` (the client's perspective, as in the UI) |
| ----- | ----- | ----- |
| **`name`** | `xmlName` | `xmlNameOtherDirection` |
| **`label`** | `id` | `text` |

So that would mean the two new items here are reversed, so it should be

```js
            xmlName: relType.name_a_b,
            xmlNameOtherDirection: relType.name_b_a,
```